app/controllers/app.rb

```diff
@@ -48,20 +47,43 @@ class App < Roda
           routing.post do
             params = JSON.parse(routing.body.read)
 
-            scopus = PaperDeep::PaperMapper.new(API_TOKEN)
+            scopus = PaperDeep::PaperMapper.new(App.config.api_key)
             scopus.search(params['keyword'])
             scopus_parse_project = scopus.parse
+
+            # puts scopus_parse_project
```

Comment:
This line seems like being used to debug. You may not keep this line on version control.